tests/test_virtual_chassis.py

```diff
@@ -136,6 +136,7 @@ def test_voq_switch(self, vct):
                 spcfg = ast.literal_eval(value)
                 assert spcfg['count'] == sp_count, "Number of systems ports configured is invalid"
 
+    @pytest.mark.skip(reason="Failing. Under investigation")
```

Comment:
Please revert back once the dependency (sonic-buildimage) PR is merged

Reply:
Yes. Will do.